Sources/ProjectDescription/ConfigGenerationOptions.swift

```diff
@@ -1,6 +1,16 @@
 extension Config {
     /// Options for project generation.
     public struct GenerationOptions: Codable, Equatable {
+        /**
+         This enum represents the targets against which Tuist will run the check for potential side effects
+         caused by static transitive dependencies.
+         */
+        public enum StaticSideEffectsWarningTargets: Codable, Equatable {
+            case all
+            case none
+            case only([String])
```

Comment:
Shouldn't this be rather `excluding([])`?
You want to exclude the one for which you know it's not a problem, but "remain protected" against new instances

Reply:
Good point. I changed it following your suggestion.